src/Markdig/Parsers/ParserList.cs

```diff
@@ -60,11 +61,11 @@ protected ParserList(IEnumerable<T> parsersArg) : base(parsersArg)
             var tempCharMap = new Dictionary<char, T[]>();
             foreach (var parser in this)
             {
-                if (parser.OpeningCharacters != null && parser.OpeningCharacters.Length != 0)
+                if (parser.OpeningCharacters is { Length: > 0 })
```

Comment:
We should stay consistent with the type of checks. The same pattern is used above in the same method but wasn't changed.

Reply:
Agreed. I had planned to follow up to apply these changes everywhere, but didn't want this PR to become to big. I replaced the remaining instances of this pattern to address this feedback now.